lib/restforce_mock/client.rb

```diff
@@ -7,6 +7,25 @@ class Client
     include ::Restforce::Concerns::API
     include RestforceMock::Sandbox
 
+    def mashify?
```

Comment:
https://github.com/execonline-inc/restforce/blob/master/lib/restforce/concerns/api.rb#L158-#L163

It returns a Restforce::Collection if set to true. The api_get method breaks if it is not set